vpr/src/power/power.cpp

```diff
@@ -147,8 +147,8 @@ static void power_usage_primitive(t_power_usage* power_usage, t_pb* pb, t_pb_gra
 
         LUT_size = pb_graph_node->num_input_pins[0];
 
-        input_probabilities = (float*)vtr::calloc(LUT_size, sizeof(float));
-        input_densities = (float*)vtr::calloc(LUT_size, sizeof(float));
+        input_probabilities = new float[LUT_size]();
```

Comment:
Again, these are calloc --> explicitly set to 0 in a loop, or verify the standard guarantees 0 initialization and comment that.